tst/testkernel/dstruct.c

```diff
@@ -2,11 +2,12 @@
  * Small program to test data structures used by the Julia GC integration.
  */
 
-#include <libgap-api.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 
+#include "libgap-api.h"
```

Comment:
Hmm, maybe `libgap-api.h` should get the same treatment as `gap_all.h`, just intended for users of the `libgap` instead of kernel extensions...